api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java

```diff
@@ -158,6 +158,11 @@ public String freeTierBillingAccountName() {
 
   public static class WgsCohortExtractionConfig {
     public String serviceAccount;
+    public String terraWorkspaceNamespace;
```

Comment:
I'm worried this could get confused in certain context with the target (researcher's) workspace. Maybe rename these two to `internalTerraWorkspace...` ?

Reply:
Sounds good